Xpirit-Vsts-Release-Terraform/terraform.ps1

```diff
@@ -134,9 +134,9 @@ function Initialize-Terraform
 
     if (-not ([string]::IsNullOrEmpty($additionalArguments)))
     {
-        $arguments = $remoteStateArguments + " $($additionalArguments.Trim()) $($extraArguments.Trim())"
```

Comment:
I think this will break the -input=false -no-color because it has to be at the end. I have an other suggestion to fix this by changing this line to check if the arguments end by dir

Reply:
hmm I haven't seen any mention that these flags must be at the very end, are you sure that is the case?